### PR TITLE
Install PIP on Debian 10

### PR DESCRIPTION
**Pull request by Tumite**
I have read and understood the Contributor's Certificate of Origin available at the end of
https://raw.githubusercontent.com/hetzneronline/community-content/master/tutorial-template.md
and I hereby certify that I meet the contribution criteria described in it.
Signed-off-by: Roland Balla balla.roland96@gmail.com

### CODE REVIEW

File: tutorials/install-pip-on-debian-10/01.en.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+
+Pip is a package management system that allows you to install Python packages. With `pip`, you can install packages from the [Python Package Index (PyPI)](https://pypi.org/) and other repositories.
+
+In this guide, we will explain how to install `pip` for both Python 2 pip and Python 3 `pip3` on Debian 10.
+
+**Prerequisites**
+
+To follow along with this tutorial, you will need a Debian 10 server with:
+
+⋅⋅*a non-root user with sudo privileges
```
Comment (n-se) [inline tutorials/install-pip-on-debian-10/01.en.md]: For what are these dots here?

Comment (Tumite) [reply tutorials/install-pip-on-debian-10/01.en.md]: Unordered sub-list, it just doesn't work.

Comment (Tumite) [reply tutorials/install-pip-on-debian-10/01.en.md]: Solved.

**Comment by Tumite** [comment]
Hi!
Have any trouble with this tutorial?

**Comment by leojakeson** [comment]
Hello Tumite, unfortunately, this tutorial is too short to be added to our Community Tutorials. In addition, linuxize.com has almost the exact same tutorial on their website, though that one is a little more detailed.

As such we won't be accepting this tutorial, sorry.

**Pull request closed without merging**